docs/topic/idle-culler.rst

```diff
@@ -13,6 +13,10 @@ JupyterHub pings the user's notebook server at certain time intervals. If no res
 is received from the server during this checks and the timeout expires, the server is
 considered to be *inactive (idle)* and will be culled.
 
+The idle culler is maintained as
+`a separate project <https://github.com/jupyterhub/jupyterhub-idle-culler>`_.
+For advanced users, the code leaves the option for direct adjustment, e.g.  to add 
+features such as purging old user data.
```

Comment:
What do you think about mentioning the `jupyterhub_config.py` escape-hatch here as a way of extending the culler config outside the `tljh-config's` scope? Maybe something like:

*The idle culler is enabled by default in TLJH and can be configured using `tljh-config`. For advanced use-cases, like purging old user data, the idle culler configuration can be extended beyond `tljh-config` options, using custom [`jupyterhub_config.py` snippets](https://tljh.jupyter.org/en/latest/topic/escape-hatch.html?highlight=escape-hatch#extending-jupyterhub-config-py)*